entanglement_forging/core/wrappers/entanglement_forged_driver.py

```diff
@@ -48,26 +64,66 @@ def __init__(
         self._num_beta = num_beta
         self._nuclear_repulsion_energy = nuclear_repulsion_energy
 
-    def run(self) -> QMolecule:
+    def run(self) -> ElectronicStructureDriverResult:
         """Returns QMolecule constructed from input data."""
-        q_molecule = QMolecule()
-        q_molecule.atom_symbol = []
-        q_molecule.atom_xyz = []
-        q_molecule.hcore = self._hcore
-        q_molecule.mo_coeff = self._mo_coeff
-        q_molecule.eri = self._eri
-        q_molecule.num_molecular_orbitals = self._mo_coeff.shape[0]
-        q_molecule.num_alpha = self._num_alpha
-        q_molecule.num_beta = self._num_beta
-        q_molecule.nuclear_repulsion_energy = self._nuclear_repulsion_energy
+        # Create ParticleNumber property
+        particle_number = ParticleNumber(
+            self._mo_coeff.shape[0], (self._num_alpha, self._num_beta)
+        )
+
```

Comment:
```suggestion
            self._mo_coeff.shape[0] * 2, (self._num_alpha, self._num_beta)
        )

```
In the `properties` framework we are consistently using the number of spin orbitals (which is twice the number of MOs). Hence you need to multiply this by 2

Reply:
Done